test/user-test.js

```diff
@@ -1,16 +1,73 @@
 import { expect } from 'chai';
 
 import User from '../src/user';
-import data from '../data/users-data';
+// import users from '../data/users-data';
 
 describe('User', function() {
-  let user;
+  let user1;
+  let user2;
   let userInfo;
   let recipe;
 
   beforeEach(function() {
-    userInfo = data.users[0];
-    user = new User(userInfo)
+    userInfo = [
```

Comment:
Let's revisit the amount of information we have in our sample dataset once we've gone through the tests. We might be able to narrow it down a bit later.